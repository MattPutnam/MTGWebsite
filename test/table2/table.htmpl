<table class="{{eval('$header'.replace(' ', '_'))}}">
    <thead>
        <tr>
            <th colspan="2">{{$header}}</th>
        </tr>
    </thead>
    <tbody>{{foreach:var=key, source=$content}}
        <tr>
            <td class="role">{{$key}}</td>
            <td class="name">{{$content->($key)}}</td>
        </tr>{{end}}
    </tbody>
</table>