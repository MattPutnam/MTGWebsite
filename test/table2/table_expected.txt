<table class="Test_Table_2">
    <thead>
        <tr>
            <th colspan="2">Test Table 2</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td class="role">Producer</td>
            <td class="name">Carol</td>
        </tr>
        <tr>
            <td class="role">Director</td>
            <td class="name">Dave</td>
        </tr>
    </tbody>
</table>